deploy_and_install_scylla_ansible_script/README.md

```diff
@@ -0,0 +1,59 @@
+General info and prerequisites
+==============================
+
+This ansible script will deploy, install and configure ScyllaDB on all supported OS.
+- RHEL7
+- Centos7
+- Ubuntu 14.04
+- Ubuntu 16.04
+- Debian 8
+
+Prerequisites:
+- Ansible 2.3
+- Python 2.7
+
+
+Instructions
+============
+
+This playbook assumes you are installing Scylla using the same disks and NIC for all nodes
```

Comment:
The example file is server_example.ini, but the READ use servers.ini
Please fix one